tests/main.cpp

```diff
@@ -712,8 +711,45 @@ TEST_CASE("getVersion") {
 }
 
 TEST_CASE("getFirstAvailableBlock") {
-  const solana::Keypair keyPair = solana::Keypair::fromFile(KEY_PAIR_FILE);
   const auto connection = solana::rpc::Connection(solana::DEVNET);
   const auto firstAvailableBlock = connection.getFirstAvailableBlock();
   CHECK_GT(firstAvailableBlock, 0);
 }
+
+TEST_CASE("getSlot") {
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  const auto slot = connection.getSlot();
+  CHECK_GT(slot, 0);
+  sleep(2);  // sleep for 2 seconds
+  const auto slot2 = connection.getSlot();
+  CHECK_GE(slot2, slot);
```

Comment:
SHould be `CHECK_GT`

Reply:
better version of this test would be to run getSlot with finalized, confirmed and getSlot with processed commitment. this way the test wouldn't need to wait for 2 seconds and you still get different slots

Reply:
Done... checks `getSlot(finalized) < getSlot(confirmed) < getSlot(processed)`